README.md

```diff
@@ -2,102 +2,102 @@
 
 > A curated list of awesome TypeDB resources.
 
-[TypeDB](https://vaticle.com) is a strongly-typed database with a rich and logical type system. TypeDB empowers you to tackle complex problems, and TypeQL is its query language
+[TypeDB](https://github.com/vaticle/typedb) is a strongly-typed database with a rich and logical type system built by [Vaticle](https://vaticle.com). TypeDB empowers you to tackle complex problems using TypeQL, TypeDB's very own query language.
 
 ## Contents
 
-- [Client Drivers (official)](#client-drivers-official)
-- [Client Drivers (community)](#client-drivers-community)
-- [IDE and editors support](#ide-and-editors-support)
```

Comment:
The client drivers are cool, but they aren't immediately exciting in the way that KGLIB and our Cyber Threat Intelligence platform are, so I've moved them down the list.